test/integration/integration_suite_test.go

```diff
@@ -102,21 +103,33 @@ var _ = BeforeSuite(func(ctx SpecContext) {
 	}
 	Expect(namespaces.Get(ctx, userNamespace, metav1.GetOptions{})).NotTo(BeNil())
 
+	By("Create ManagedClusterSetBinding")
+	err := applyManagedClusterSetBinding(ctx)
+	Expect(err).ToNot(HaveOccurred())
+
 	By("Setting up GitOps user")
 	common.GitOpsUserSetup(ctx, &gitopsUser)
 })
 
 var _ = AfterSuite(func(ctx SpecContext) {
-	By("Delete Namespace if needed")
+	if userNamespace != "open-cluster-management-global-set" {
+		By("Delete Namespace if needed")
+		_, err := common.OcHub(
+			"delete", "namespace", userNamespace,
+			"--ignore-not-found",
+		)
+		Expect(err).ToNot(HaveOccurred())
+	}
+
 	_, err := common.OcHub(
-		"delete", "namespace", userNamespace,
-		"--ignore-not-found",
+		"delete", "pod", "default",
+		"pod-that-does-not-exist", "--ignore-not-found",
 	)
 	Expect(err).ToNot(HaveOccurred())
 
 	_, err = common.OcHub(
```

Comment:
This should also be in the `if userNamespace != "open-cluster-management-global-set" {` block.

Reply:
sorry Could you elaborate this?